grammars/css.cson

```diff
@@ -90,14 +99,14 @@
     ]
   }
   {
-    'begin': '(?=^\\s*@media(\\s+|$))'
+    'begin': '(?=@media(\\s+|$))'
```

Comment:
We'd need to use `@media(?:\\/\\*.*?\\*\\/)?(\\s+|$)` but I'm afraid the grammar might become overly complicated (we need to do this change in several places).